doc/user-guide/jupyter/trame.rst

```diff
@@ -0,0 +1,171 @@
+.. _trame_jupyter:
+
+Trame Jupyter backend for PyVista
+---------------------------------
+
+PyVista has the ability to display fully featured plots within a
+Jupyter environment using `Trame <https://kitware.github.io/trame/index.html>`_.
+We provide mechanisms to pair PyVista and Trame so that PyVista plotters
+can be used in a web context with both server- and client-side rendering.
+
+The server-side rendering mode of the Trame backend works by streaming the
+current render window to a canvas within Jupyter and then passing any user
+actions from the canvas back to the VTK render window (this is done under
+the hood by the ``vtkRemoteView`` in ``trame-vtk``.
+
+For example, both sections of code will display an interactive canvas
+within Jupyter:
+
+.. code:: python
+
+    import pyvista as pv
+    sphere = pv.Sphere()
+
+    # short example
+    sphere.plot(jupyter_backend='trame')
+
+    # long example
+    plotter = pv.Plotter(notebook=True)
+    plotter.add_mesh(sphere)
+    plotter.show(jupyter_backend='trame')
+
+For convenience, you can enable ``trame`` by default with:
+
+.. note::
+    It is critical to ``await`` the call to :func:`set_jupyter_backend() <pyvista.set_jupyter_backend>` when using Trame in Jupyter.
+
+.. code:: python
+
+    import pyvista as pv
+    await pv.set_jupyter_backend('trame')
```

Comment:
```suggestion
.. code:: python

    import pyvista as pv
    await pv.set_jupyter_backend('trame')

.. note::
    It is critical to ``await`` the call to :func:`set_jupyter_backend() <pyvista.set_jupyter_backend>` when using Trame in Jupyter.
```

Reply:
"Notes" seem to make more sense after calling something to explain something than beforehand.